docs/check-redirects/pnpm-workspace.yaml

```diff
@@ -0,0 +1,6 @@
+# Because of a bug pnpm, we need to have this "empty" pnpm-workspace.yaml file.
+# Without it, dependencies from the local package.json were not installed.
```

Comment:
Don't you mean that the deps were added into the main lockfile?

Reply:
Ah no, actually `docs/check-redirects/package.json` was simply ignored. I clarified the comment.

Reply:
Aaaah, I totally misunderstood your comment here. https://github.com/gravitational/teleport/pull/44857#discussion_r1698789513

I thought that despite adding a separate package.json, the deps from it were included in the main lockfile. But I see now that `pnpm i` from `docs/check-redirects` ignores the "local" package.json and installs deps from the workspace.

Tbh, this behavior doesn't seem that unreasonable to me. Of course, pnpm could have some smarter heuristics when it comes to deciding what package is and isn't a part of a workspace, but this could lead to other edge cases I suppose. Having to create a separate `pnpm-workspace.yaml` seems just fine to me.

Reply:
Hmm makes sense, I rephrased the comment one more time.